Guardian Abilities Attempt

Psychonator17 opened the pull request:
In case it didn't go through, here's my version of "ability_support.json" which _should_ give the Guardian (and Elder Guardian) swimming and hostile abilities. Is is possible to integrate this into the current version? And if there's something wrong with the formatting, please address it.

[Copy of Custom Ability Sheet.txt](https://github.com/iChun/Morph/files/4720350/Copy.of.Custom.Ability.Sheet.txt)

iChun commented:
Sorry, I've only seen this now. Are you pulling from an old fork of the mod? There's a bagazillion commits

Psychonator17 commented:
> Sorry, I've only seen this now. Are you pulling from an old fork of the mod? There's a bagazillion commits

Not sure. I know that I downloaded a copy of the original ability_support.jsn and modified it to include Guardian/Edler Guardian abilities. Speaking of which, I'd learned more about how to add modifications and updated it a bit.

[ability_support.txt](https://github.com/iChun/Morph/files/6290856/ability_support.txt)

iChun commented:
Can you redo the pull request? The branch you're trying to merge with is severely outdated with a tonne of commits. Ideally it should be just a couple of changes in just ability_support.json. I'm going to be closing this for now.
